libs/util/code-gen-utils/src/lib/files.ts

```diff
@@ -4,21 +4,23 @@ import { memoize } from "lodash";
 import { format } from "prettier";
 import { parse } from "./parse/main";
 
-export const formatCode = (path: string, code: string): string => {
-  if (path.endsWith(".ts") || path.endsWith(".tsx")) {
-    return format(code, { parser: "typescript" });
-  }
-  if (path.endsWith(".json")) {
-    return format(code, { parser: "json" });
-  }
-  if (path.endsWith(".yml") || path.endsWith(".yaml")) {
-    return format(code, { parser: "yaml" });
-  }
-  if (path.endsWith(".md")) {
-    return format(code, { parser: "markdown" });
-  }
-  if (path.endsWith(".graphql")) {
-    return format(code, { parser: "graphql" });
+export const formatCode = (code: string, path?: string): string => {
+  if (path) {
```

Comment:
This is partially backward compatible since if no path was given, it will not format at all, compared to the original code.
Perhaps we can , if no path was specified, try to format the code as "typescript" and catch any exception and return the give code as-is on exception? Since most modules are ts/tsx, this will ensure formatting on at least most files. WDYT?